README.md

```diff
@@ -0,0 +1,89 @@
+Django-locking
+==============
+
+About
+-----
+
+`django-locking` provides a system that makes concurrent editing impossible, and informs users of what other users are working on and for how long that content will remain locked. Users can still read locked content, but cannot modify or save it.
+
+Requirements
+-------------
+
+* Django 2.2
+* Python 3.6 - 3.8
+
+Installation
+------------
+
+1. Download and install django-locking.
+
+    ```console
+    $ pip install git+git://github.com/unt-libraries/django-locking.git
+    ```
+
+2. Add app to INSTALLED_APPS.
+
+    ```python
+    INSTALLED_APPS = (
+        'locking',
+    )
+    ```
+
+Developing
+----------
+
+1. Clone the git repository:
+
+    ```python
+        git clone https://github.com/unt-libraries/django-locking.git
+    ```
+
+2. Navigate into the cloned repository:
+
+    ```python
+        cd django-locking
+    ```
+
+3. Install the requirements (preferably in a virtual environment):
+
+    ```python
+        python setup.py install
+    ```
+
+4. Run the migrations:
+
+    ```python
+        django-admin migrate
+    ```
+
+5. Create a superuser so you can log into the admin interface
+
+    ```python
+        django-admin createsuperuser
+    ```
+
+6. Start the test server:
+
+    ```python
+        django-admin runserver
+    ```
+
+Testing
+-------
+
+1. Run the tests (inside the virtualenv):
+
+    ```python
+        django-admin test
+    ```
+
+License
+-------
+
+See LICENSE
+
+Contributors
+------------
+
+* [Gio Gottardi](https://github.com/somexpert)
+* [Madhulika Bayyavarapu](https://github.com/madhulika95b)
```

Comment:
Since this is forked from debrouwere, we should list him too. It looks like he was the only contributor on that project.